beacon-chain/execution/engine_client.go

```diff
@@ -491,9 +491,6 @@ func (s *Service) HeaderByNumber(ctx context.Context, number *big.Int) (*types.H
 
 // GetPayloadBodiesByHash returns the relevant payload bodies for the provided block hash.
 func (s *Service) GetPayloadBodiesByHash(ctx context.Context, executionBlockHashes []common.Hash) ([]*pb.ExecutionPayloadBodyV1, error) {
-	if !features.Get().EnableOptionalEngineMethods {
```

Comment:
We use this feature flag in a lot of other places. Do we want to remove this one all the way across the codebase too?

Reply:
I looked at the other places and we do not use those particular methods anywhere in runtime, they are simply implemented. I can remove the flag in these other places as its kind of pointless to have it.